### PR TITLE
Submitit ask-tell tutorial

### PR DESCRIPTION
**Pull request by mhavasi**
Adds a tutorial that shows how to use the AxClient to schedule jobs and tune parameters on a Slurm cluster via SubmitIt.

Code changes:
1. The tutorial notebook
2. Linking the tutorial notebook for the website. (index.js, tutorials.json)

Testing:
Q: How can I make sure that this tutorial shows up correctly on the Ax doc site?

### CODE REVIEW

**Comment by mpolson64** [comment]
Looks great! A couple notes (that I will just leave here since commenting on a .ipynb is annoying):
* I can handle making sure this renders properly on the website. It's a little easier for me to do this once things have landed in fbsource so let's just get this in as is for now 
* Let's use `exp_to_df(ax_client.experiment)` instead of `ax_client.generation_strategy.trials_as_df` -- marginally better UX imo
* Can we add a cell where we plot the response surface and optimization trace like we do here: https://ax.dev/tutorials/gpei_hartmann_service.html#6.-Plot-the-response-surface-and-optimization-trace ?
* All the tutorials run locally during our testing, so can we comment out `cluster='slurm'` and replace with `cluster='local'` and a comment explaining that the cluster is set to local for testing purposes

**Comment by mpolson64** [comment]
One last thing: no need to write the covariance since the demo problem only has one metric

**Comment by mhavasi** [comment]
Thanks so much for the comments! I fixed the nits and I did another grammar pass.

File: tutorials/submitit.ipynb [hunk 1]
```diff
@@ -0,0 +1,7550 @@
...
+    "    return {\"result\": (x - 3)**2 + (y - 4)**2}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note 1: SubmitIt's [CommandFunction](https://github.com/facebookincubator/submitit/blob/main/docs/examples.md#working-with-commands) allows you to define commands to run on the node and then redirects the standard output.\n",
+    "\n",
+    "Note 2: If you are using Hydra to manage configs, SubmitIt also has [Hydra integration](https://hydra.cc/docs/plugins/submitit_launcher/).\n",
```
Comment (mshvartsman) [inline tutorials/submitit.ipynb:58]: This is true, but this tutorial doesn't cover overriding hydra params using Ax trial configurations. So maybe should either add an example of that (here or to another tutorial), or maybe should just remove this note?

File: tutorials/submitit.ipynb [hunk 2]
```diff
@@ -0,0 +1,7550 @@
...
+    "Other commonly used [parameters types](https://ax.dev/docs/glossary.html#parameter) include `choice` parameters and `fixed` parameters. \n",
+    "\n",
+    "Tip 1: you can specify additional information for parameters such as `log_scale`, if a parameter operates at a log-scale and `is_ordered` for choice parameters that have a meaningful ordering.\n",
+    "\n",
+    "Tip 2: Ax is an excellent choice for multi-objective optimization problems when there are multiple competing objectives and the goal is to find all Pareto-optimal solutions.\n",
+    "\n",
+    "Tip 3: One can define constraints on both the parameters and the outcome.\n",
+    "\n",
+    "## Setting up SubmitIt\n",
+    "We'll use SubmitIt's `AutoExecutor` for this example. We start by initializing an `AutoExecutor`, and setting a few commonly used parameters. The full list of parameters is available here."
```
[the remark this answers is not in the document]
Comment (mhavasi) [reply tutorials/submitit.ipynb:112]: Removed sentence because I mention the commonly used parameters below.

File: tutorials/submitit.ipynb (hunk 1 above)
Comment (mhavasi) [reply tutorials/submitit.ipynb:58]: Removed note, you are right that it doesn't fit here.

**Comment by Balandat** [comment]
This is great, thanks for adding this!

**Pull request closed without merging**